config/notifiers.go

```diff
@@ -92,6 +92,17 @@ var (
 		// TODO: Add a details field with all the alerts.
 	}
 
+	// DefaultVictorOpsConfig defines default values for VictorOps configurations.
+	DefaultVictorOpsConfig = VictorOpsConfig{
+		NotifierConfig: NotifierConfig{
+			VSendResolved: true,
+		},
+		MessageType:  `{{ if eq .Status "firing" }}CRITICAL{{ else }}RECOVERY{{ end }}`,
```

Comment:
Does Victorops treat messages differently depending on this? We may wish to hardcode resolved notifications to send a recovery message.